src/Sulu/Component/Content/Export/WebspaceExport.php

```diff
@@ -299,6 +299,8 @@ protected function getDocumentsQueryString($webspaceKey, $uuid = null, $nodes =
 
         $queryString = 'SELECT * FROM [nt:unstructured] AS a WHERE ' . \implode(' AND ', $where);
 
+        $queryString .= ' ORDER BY [jcr:path] ASC';
+
```

Comment:
This avoids error on postgres as it not returning the result in the same order as mysql when using doctrine dbal adapter